base-image/build/conf/machine/generic-armv5.conf

```diff
@@ -0,0 +1,11 @@
+#@TYPE: Machine
+#@NAME: Generic armv5te machine
+#@DESCRIPTION: Machine configuration for armv5te based boards
+
+require conf/machine/include/qemu.inc
+require conf/machine/include/arm/arch-armv5.inc
+
+KERNEL_IMAGETYPE = "zImage"
+
+SERIAL_CONSOLES = "115200;ttyAMA0 115200;ttyAMA1"
```

Comment:
Just checked, these variables are kernel variables define type of kernel and the serial consoles (TTYs) to enable using getty. So can be removed.